src/Pact/Native/Ops.hs

```diff
@@ -155,7 +155,7 @@ lteDef = defCmp "<=" (cmp (`elem` [LT,EQ]))
 eqDef :: NativeDef
 eqDef = defRNative "=" (eq id) eqTy
   ["(= [1 2 3] [1 2 3])", "(= 'foo \"foo\")", "(= { 'a: 2 } { 'a: 2})"]
-  "True if X equals Y."
+  "Compare alike terms for equality. Returns true if X equals Y."
```

Comment:
This needs to document the operational behavior more to indicate that it will fail on type mismatch OR if types are not value types.